components/user_settings/sidebar/user_settings_sidebar.tsx

```diff
@@ -859,7 +859,7 @@ export default class UserSettingsSidebar extends React.PureComponent<UserSetting
 
         const channelOrganizationSection = (showChannelOrganization && channelSidebarOrganizationDisabled) ? this.renderChannelOrganizationSection() : null;
         const channelSidebarOrganizationSection = showChannelSidebarOrganization ? this.renderChannelSidebarOrganizationSection() : null;
-        const autoCloseDMSection = showUnusedOption ? this.renderAutoCloseDMSection() : null;
+        const autoCloseDMSection = showUnusedOption && !channelSidebarOrganizationDisabled ? this.renderAutoCloseDMSection() : null;
```

Comment:
Should this maybe be added wherever `showUnusedOption` comes from?

Reply:
its coming from 'config.CloseUnusedDirectMessages === 'true'' and the naming is not explicit that the sidebar organisation is disabled unless we change the naming or create a function for it. I am 1/5 on this because it seems clearer to me and we already have the 'channelSidebarOrganizationDisabled' explicitly stored in the file, but I can create a function if you like.

Reply:
Fair enough. I think `channelSidebarOrganizationDisabled` could be used to generate `showUnusedOption` since it's important for whether or not to show that section, but I see your point about `channelSidebarOrganizationDisabled` being used elsewhere as well.